modules/cluster-logging-elasticsearch-scaledown.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-log-store.adoc
+
+[id="cluster-logging-elasticsearch-scaledown_{context}"]
+= Scaling down Elasticsearch nodes
+
+Reducing the number of Elasticsearch nodes in your cluster can result in data loss or Elasticsearch performance degradation. 
+
+If you scale down, you should scale down by one node at a time and allow the cluster to re-balance the shards and replicas. After the Elasticsearch status returns to green, you can scale down by another node. 
```

Comment:
I'm not sure how "returns to green" will translate. Perhaps you can say something like "After the Elasticsearch status shows a successful scale down, indicated by a green status on the dashboard, ..."   or something like that.